addons/csv-data-importer/import_plugin.gd

```diff
@@ -127,8 +129,8 @@ func _import(source_file, save_path, options, platform_variants, gen_files):
 			printerr("Can't find header in empty file")
 			return ERR_PARSE_ERROR
 
-		var headers = lines[0]
-		for i in range(1, lines.size()):
+		var headers = lines[options.headers_line]
```

Comment:
This access needs to be protected by a boundary check. If a out-of-boundary line is specified, it should `printerr()` and return `ERR_PARSE_ERROR` I think.

Sorry for the late review.